core/storage/base_model/gae_models.py

```diff
@@ -389,6 +490,16 @@ def get_version(cls, model_instance_id, version_number):
 
         The snapshot content is used to populate this model instance. The
         snapshot metadata is not used.
+
+        Args:
+            model_instance_id: str.
+            version_number: int.
+
+        Returns:
+            VersionModel.
```

Comment:
VersionedModel, here and below. Also add docstring to explain what the return value represents.